### ISSUE TITLE
usage

### ISSUE THREAD

**Issue by Maikuraky** [noureldien/timeception#10]
Hey, can you give me a detailed usage?
For example: which data set to download, how to divide the data set into training set and test set, where to put the data, etc.
